pkg/master/master.go

```diff
@@ -147,6 +147,9 @@ func (c *Config) Complete() completedConfig {
 		c.EndpointReconcilerConfig.Reconciler = NewMasterCountEndpointReconciler(c.GenericConfig.MasterCount, endpointClient)
 	}
 
+	// this has always been hardcoded true in the past
+	c.GenericConfig.EnableMetrics = true
```

Comment:
any reason to do that in `Complete` and not explicitly where `Config` is created? I would keep `Complete` minimal.